bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/messages/CommandPayload.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.paradoxalarm.internal.communication.messages;
+
+import java.nio.ByteBuffer;
+
+/**
+ * The {@link CommandPayload} Class that structures the payload for partition commands.
+ *
+ * @author Konstantin Polihronov - Initial contribution
+ */
+public class CommandPayload implements IPayload {
```

Comment:
Done. Not hard at all. I've added it to the enums also. Does it make sense for them?

Reply:
For all new bindings `@NonNullByDefault` is required for all non-DTO classes. Existing bindings are only required to add `@NonNullByDefault` to new classes added to them, and even then, only if it doesn't cause an excessive amount of additional changes to a PR since we don't want to punish new contributors with trying to fix/refactor old to code to adhere to the new policy.